scipy/stats/_distn_infrastructure.py

```diff
@@ -100,8 +100,10 @@
     (Differential) entropy of the RV.
 """
 _doc_fit = """\
-fit(data, %(shapes)s, loc=0, scale=1)
+fit(data)
     Parameter estimates for generic data.
+    See `scipy.stats.rv_continuous.fit <https://docs.scipy.org/doc/scipy/reference/generated/scipy.stats.rv_continuous.fit.html#scipy.stats.rv_continuous.fit>`__ for detailed documentation of the
```

Comment:
Much better. This is non-blocking, but I was hoping we could use some sort of internal cross-reference instead of an explicit URL. Do you know how to do that? I have only used `:ref:` when I am editing `.rst` files themselves, no docstrings, and it's unclear to me how exactly attribute names in single backticks sometimes turn into links.

Reply:
@mdhaber Can I take this internal cross-reference another PR? I'm afraid of breaking this one if I continue here

Reply:
OK.